dbt_subprojects/daily_spellbook/models/_metrics/fees/chains/solana/metrics_solana_gas_fees_daily.sql

```diff
@@ -13,48 +15,64 @@ with fees as (
     select
         blockchain
         , block_date
+        , tx_from as address
         , sum(tx_fee_usd) as gas_fees_usd
     from
         {{ source('gas', 'fees') }}
-    where blockchain = 'solana'
-    {% if is_incremental() %}
-    and
-        {{ incremental_predicate('block_date') }}
-    {% endif %}
+    where
+        blockchain = '{{ blockchain }}'
+        {% if is_incremental() or true %}
+        and {{ incremental_predicate('block_date') }}
+        {% endif %}
     group by
         blockchain
         , block_date
+        , tx_from
 ), solana_vote_fees as (
     -- solana vote fees are stored in a different spell due to data volume & lack of value-add for materializing the fee breakdown
     select
         blockchain
         , block_date
+        , from_base58(signer) as address
         , sum(tx_fee_usd) as gas_fees_usd
     from
         {{ source('gas_solana', 'vote_fees') }}
-    {% if is_incremental() %}
     where
-        {{ incremental_predicate('block_date') }}
-    {% endif %}
+        1 = 1
+        {% if is_incremental() or true %}
+        and {{ incremental_predicate('block_date') }}
+        {% endif %}
     group by
         blockchain
         , block_date
+        , from_base58(signer)
 ), combined_fees as (
     select
-        fees.blockchain
-        , fees.block_date
-        , fees.gas_fees_usd + coalesce(solana_vote_fees.gas_fees_usd, 0) as gas_fees_usd
+        coalesce(fees.blockchain, solana_vote_fees.blockchain) as blockchain
+        , coalesce(fees.block_date, solana_vote_fees.block_date) as block_date
+        , coalesce(fees.address, solana_vote_fees.address) as address
+        , coalesce(fees.gas_fees_usd, 0) + coalesce(solana_vote_fees.gas_fees_usd, 0) as gas_fees_usd
     from
         fees
-    left join
+    full outer join
```

Comment:
switched from left join to full outer join, as not all addresses which had voted also exist in transactions.

query to test, notice the comment on row counts for context:
```
with
    tx as (
        select
            blockchain,
            block_date,
            tx_from as address
        from
            gas.fees
        where
            blockchain = 'solana'
            and block_date >= date_trunc('day', now() - interval '3' day)
        group by
            blockchain,
            block_date,
            tx_from
    ),
    vote as (
        select
            blockchain,
            block_date,
            from_base58 (signer) as address
        from
            gas_solana.vote_fees
        where
            blockchain = 'solana'
            and block_date >= date_trunc('day', now() - interval '3' day)
        group by
            blockchain,
            block_date,
            signer
    )
    --14,867,450 total tx
    --5,799 total vote (1,969 of those vote addresses don't appear in regular tx fees)
select
    count(1)
from
    tx
    right join vote on tx.blockchain = vote.blockchain
    and tx.block_date = vote.block_date
    and tx.address = vote.address
where
    tx.blockchain is null
```